packages/cashier/src/Stores/Cashier/cashier-store.js

```diff
@@ -160,6 +161,10 @@ export default class CashierStore extends BaseStore {
             root_store: this.root_store,
             WS: this.WS,
         });
+
+        this.transaction_history = new TransactionHistoryStore({
```

Comment:
Just a suggestion, Since it's just one input you can pass it as a single variable not an object.